telescope/external.py

```diff
@@ -54,96 +55,100 @@ class BigQueryCommunicationError(Exception):
   to retrieve status from BigQuery.
   """
 
-  def __init__(self, message, cause):
-    self.cause = cause
-    super(BigQueryCommunicationError, self).__init__(
-        '%s (%s)' % (message, self.cause))
+    def __init__(self, message, cause):
+        self.cause = cause
+        super(BigQueryCommunicationError, self).__init__(
+            '%s (%s)' % (message, self.cause))
 
 
 class TableDoesNotExist(Exception):
 
-  def __init__(self):
-    Exception.__init__(self)
+    def __init__(self):
+        Exception.__init__(self)
 
 
 class APIConfigError(Exception):
 
-  def __init__(self):
-    Exception.__init__(self)
+    def __init__(self):
+        Exception.__init__(self)
 
 
 class GoogleAPIAuthConfig:
-  """ Google API requires an object with preferences for logging and
+    """ Google API requires an object with preferences for logging and
       authentication. Rather than pass with argparse, for now we manually
       build the default configurations to pass directly. Not guaranteed to
       stay in the future.
 
   """
-  logging_level = 'ERROR'
-  noauth_local_webserver = False
-  auth_host_port = [8080, 8090]
-  auth_host_name = 'localhost'
+    logging_level = 'ERROR'
+    noauth_local_webserver = False
+    auth_host_port = [8080, 8090]
+    auth_host_name = 'localhost'
 
 
 class GoogleAPIAuth:
 
-  def __init__(self, credentials_filepath, is_headless=False):
-    self.logger = logging.getLogger('telescope')
-    self.credentials_filepath = credentials_filepath
-    self._set_headless_mode(is_headless)
-    self.project_id = self._find_project_id_opportunistically()
+    def __init__(self, credentials_filepath, is_headless=False):
+        self.logger = logging.getLogger('telescope')
+        self.credentials_filepath = credentials_filepath
+        self._set_headless_mode(is_headless)
+        self.project_id = self._find_project_id_opportunistically()
 
-  def _set_headless_mode(self, is_headless):
-    GoogleAPIAuthConfig.noauth_local_webserver = is_headless
+    def _set_headless_mode(self, is_headless):
+        GoogleAPIAuthConfig.noauth_local_webserver = is_headless
 
-  def authenticate_with_google(self):
+    def authenticate_with_google(self):
 
-    flow = flow_from_clientsecrets(
-        os.path.join(os.path.dirname(__file__),
-                     'resources/client_secrets.json'),
-        scope='https://www.googleapis.com/auth/bigquery')
-    storage = Storage(self.credentials_filepath)
-    credentials = storage.get()
+        flow = flow_from_clientsecrets(
+            os.path.join(os.path.dirname(__file__),
+                         'resources/client_secrets.json'),
```

Comment:
If the first parameter is on the same line as the left paren, we indent subsequent parameters to align with the first parameter. If the first param starts on the next line, we add an indent from the previous line:

```
do_something('/somepath/foo/bar/baz/longfilename.txt',
             '/otherpath/baz/boo/bar/longfilename.txt')

do_other_thing(
    '/superlongpaththatdoesntevenfitonthesamelineasfuncname/baz.txt',
    'cats.txt')
```